server/routes/index.ts

```diff
@@ -30,12 +30,115 @@ export default function routes(): Router {
       completedHours: 0,
       totalHours: 100,
     }
-    res.render('pages/pop/progress', { progress })
+
+    // const appointment = ""; // if no appointment
+
+    const appointment = {
+      title: 'Community Garden Maintenance',
+      date: 'Friday 15 March 2024',
+      time: '09:00',
+      location: '123 Garden Street, London SE1 7TH',
+    }
+    res.render('pages/pop/progress', { progress, appointment })
   })
 
   get('/conditions', async (req, res, next) => {
     res.render('pages/pop/conditions')
   })
 
+  get('#', async (req, res, next) => {
+    const progress = {
+      completedHours: 0,
+      totalHours: 100,
+    }
+    res.render('#', { progress })
+  })
+
+  get('/appointment-type', async (req, res, next) => {
+    const appointmentTypes = [
+      {
+        title: 'Meeting with Probation Officer',
+        description: 'Schedule a meeting with your probation officer to discuss your progress and any concerns.',
+      },
+      {
+        title: 'Unpaid Work',
+        description: 'Book an unpaid work placement session to fulfill your community service requirements.',
+      },
+      {
+        title: 'Rehabilitation Activity',
+        description: 'Participate in activities designed to help with rehabilitation and reduce reoffending.',
+      },
+      {
+        title: 'Behaviour Programmes',
+        description: 'Attend structured programmes aimed at addressing specific behavioral issues.',
+      },
+      {
+        title: 'Interventions',
+        description: 'Access specialized interventions tailored to your individual needs and circumstances.',
+      },
+    ]
+    res.render('pages/pop/appointment-type', { appointmentTypes })
+  })
+
+  get('/appointments', async (req, res, next) => {
+    const appointments = [
+      {
+        date: 'Thursday 27 Feb 2025',
```

Comment:
These times and dates need rationalising - not all appointments can be on one day and the london / manchester split doesn't make sense. We need a fake gardening project in manchester I would suggest